src/encoded/static/components/search.js

```diff
@@ -700,12 +705,89 @@ TextFilter.propTypes = {
 // Displays the entire list of facets. It contains a number of <Facet> components.
 export const FacetList = (props) => {
     const { context, facets, filters, mode, orientation, hideTextFilter, addClasses, docTypeTitleSuffix, supressTitle, onFilter } = props;
+
+    const [expandedFacets, setExpandFacets] = React.useState(new Set());
+
+    // Get facets from storage that need to be expanded
+    React.useEffect(() => {
+        const facetsStorage = sessionStorage.getItem(FACET_STORAGE);
+        const facetList = new Set(facetsStorage ? facetsStorage.split(',') : []);
+
+        sessionStorage.setItem(FACET_STORAGE, facetList.size !== 0 ? [...facetList].join(',') : []);
```

Comment:
I think we talked about this before, but I don’t remember the reason. Why `[...facetList].join(',')` instead of `facetList.join(',')`?

Reply:
facetList is a Set and it does not have join-method. I used [...faceList] to convert faceList to an array then use .join on that. Is that acceptable to you?

Reply:
Oh, I see. Yeah, that seems good then.